scripts/buildtools/commands/verify-package.js

```diff
@@ -3,8 +3,8 @@ const { readFileSync } = require('fs');
 
 const KB = 1024;
 const MB = 1024 * 1024;
-const EXPECTED_PACKAGE_SIZE = 13.35 * MB;
-const EXPECTED_PACKAGE_FILES = 2200;
+const EXPECTED_PACKAGE_SIZE = 45 * MB;
+const EXPECTED_PACKAGE_FILES = 21000;
```

Comment:
I had to increase these package and file size/counts because of the new results generated by `npm pack` with Gen 3 bundled. Looking to see get some eyes on this to determine if this is acceptable?

Reply:
I think these numbers are the total size of _all_ of the bundle outputs? If that's the case then I would expect both of these to increase